lua/cosmo/core/sv_http.lua

```diff
@@ -27,6 +27,28 @@ function HTTP_CLIENT:SetAuthorizationToken(token)
     self.Defaults.Headers["Authorization"] = "Bearer " .. token
 end
 
+local function httpSuccess(code, body, headers)
+    if code != 200 then
```

Comment:
Success doesn't always imply it's a 200, for example the API returns 204 aswell for the update endpoints